<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_project</class>
 <widget class="QWidget" name="new_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="proname_lb">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="proinfor_lb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="protype_lb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="proprice_lb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品价格：￥</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronumber_lb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proname_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="proinfor_text">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>550</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交商品</string>
   </property>
  </widget>
  <widget class="QComboBox" name="protype_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
   <item>
    <property name="text">
     <string>图书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>衣服</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>食品</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="proprice_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pronumber_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QLabel" name="upSuccess">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交成功！</string>
   </property>
  </widget>
  <widget class="QLabel" name="upFail">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交失败！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
